### PR TITLE
6 anomalies

### PR DESCRIPTION
**Pull request by ClementChaumel**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -19,6 +19,8 @@ android {
     }
 }
 
+
+
```
Comment (tbille) [inline app/build.gradle]: please remove those 2 lines

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -23,5 +37,4 @@
             </intent-filter>
         </activity>
     </application>
-    <uses-permission android:name="android.permission.INTERNET"/>
```
Comment (tbille) [inline app/src/main/AndroidManifest.xml:26]: please keep this line

File: app/src/main/java/com/m2dl/mapus/mapus/AnomalieFragment.java [hunk 3]
```diff
@@ -1,28 +1,39 @@
 package com.m2dl.mapus.mapus;
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/AnomalieFragment.java]: remove one line

File: app/src/main/java/com/m2dl/mapus/mapus/AnomalieFragment.java [hunk 4]
```diff
@@ -1,28 +1,39 @@
...
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.RadioButton;
+import android.widget.RadioGroup;
+import android.widget.Toast;
+
+import com.m2dl.mapus.mapus.firebase.AnomalieDataSource;
+import com.m2dl.mapus.mapus.model.Anomalie;
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/AnomalieFragment.java]: remove one line

File: app/src/main/java/com/m2dl/mapus/mapus/AnomalieFragment.java [hunk 5]
```diff
@@ -60,7 +71,45 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
...
+        Button retour = (Button) view.findViewById(R.id.anomaly_cancel);
+        retour.setOnClickListener(this);
+        Button envoyer = (Button) view.findViewById(R.id.anomaly_confirm);
+        envoyer.setOnClickListener(this);
+        Uri uri = Uri.parse(imgUri);
+        photo.setImageURI(uri);
+        return view;
     }
 
+
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/AnomalieFragment.java]: remove line

File: app/src/main/java/com/m2dl/mapus/mapus/MainActivity.java [hunk 6]
```diff
@@ -78,8 +110,9 @@ public boolean onNavigationItemSelected(MenuItem item) {
                 changeFragment(geolocFragment);
                 break;
             case R.id.nav_anomalie:
-                AnomalieFragment anomalieFragment = AnomalieFragment.newInstance("Var 1", "Var 2");
-                changeFragment(anomalieFragment);
+                takePictureAnomaly();
+                //AnomalieFragment anomalieFragment = AnomalieFragment.newInstance("Var 1", "Var 2");
+                //changeFragment(anomalieFragment);
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/MainActivity.java]: remove comments

File: app/src/main/java/com/m2dl/mapus/mapus/MainActivity.java [hunk 7]
```diff
@@ -119,4 +152,90 @@ private void changeFragment(Fragment fragment) {
...
+    public void onRequestPermissionsResult(int requestCode,
+                                           String permissions[], int[] grantResults) {
+        switch (requestCode) {
+            case 1: {
+                if (grantResults.length > 0
+                        && grantResults[0] == PackageManager.PERMISSION_GRANTED) {
+
+                    Log.d("CAMERA", "onRequestPermissionsResult: PERMISSION ACCORDEE");
+
+
```
Comment (tbille) [inline app/src/main/java/com/m2dl/mapus/mapus/MainActivity.java]: remove line

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (ClementChaumel) [reply app/src/main/AndroidManifest.xml:26]: uses-permission should be above application block

**Review by tbille — APPROVED** [review]

**Pull request merged**